### PR TITLE
Typo fixes in docs

### PR DESCRIPTION
**Pull request by BOHEUS**

### CODE REVIEW

File: packages/twenty-website/src/content/developers/frontend-development/hotkeys.mdx [hunk 1]
```diff
@@ -144,7 +144,7 @@ As an example, the hotkey scopes for each page are defined in the `PageHotkeySco
 ```tsx
 export enum PageHotkeyScope {
   Settings = 'settings',
-  CreateWokspace = 'create-workspace',
+  CreateWorkspace = 'create-workspace',
```
Comment (Weiko) [inline packages/twenty-website/src/content/developers/frontend-development/hotkeys.mdx:147]: I guess we have the same typo in twenty-front

**Review by Weiko — APPROVED** [review]
LGTM, thank you @BOHEUS

File: packages/twenty-website/src/content/developers/frontend-development/hotkeys.mdx (hunk 1 above)
Comment (BOHEUS) [reply packages/twenty-website/src/content/developers/frontend-development/hotkeys.mdx:147]: I'll check this

Comment (BOHEUS) [reply packages/twenty-website/src/content/developers/frontend-development/hotkeys.mdx:147]: There is, I'll create a PR with more typo fixes soon

**Pull request merged**